Tests/Integration/Service/Tika/ServerServiceTest.php

```diff
@@ -101,26 +102,28 @@ protected function setUpDocumentsStorageMock()
         $this->testDocumentsPath = ExtensionManagementUtility::extPath('tika')
             . 'Tests/TestDocuments/';
 
-        $documentsDriver = $this->createDriverFixture(array(
-            'basePath' => $this->testDocumentsPath,
-            'caseSensitive' => true
-        ));
+        $documentsDriver = $this->createDriverFixture([
+                'basePath' => $this->testDocumentsPath,
+                'caseSensitive' => true
+            ]
+        );
```

Comment:
thus this closing bracket should be on the previous line together with the closing square bracket, keeping the same indentation levels.

There were a couple more of these. I went ahead and fixed them.